### PR TITLE
Update baseline images for plot3d tests

### PR DESCRIPTION
**Pull request by michaelgrund**
**Description of proposed changes**

Address #2961.

Only minor changes due to gs version bumping.

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If wrapping a new module, open a 'Wrap new GMT module' issue and submit reasonably-sized PRs.
- [ ] If adding new functionality, add an example to docstrings or tutorials.
- [ ] Use underscores (not hyphens) in names of Python files and directories.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash command is:

- `/format`: automatically format and lint the code

### CODE REVIEW

**Comment by seisman** [comment]
> added | pygmt/tests/baseline/test_plot3d_ogrgmt_file_multipoint_default_style_Path.png
> -- | --
> added | pygmt/tests/baseline/test_plot3d_ogrgmt_file_multipoint_default_style_str.png
> 

These two images are the same as `test_plot3d_ogrgmt_file_multipoint_default_style.png` and should not be added.

**Comment by michaelgrund** [comment]
> > added
> > pygmt/tests/baseline/test_plot3d_ogrgmt_file_multipoint_default_style_Path.png
> > 
> > 
> > 
> > 
> > added
> > pygmt/tests/baseline/test_plot3d_ogrgmt_file_multipoint_default_style_str.png
> 
> These two images are the same as `test_plot3d_ogrgmt_file_multipoint_default_style.png` and should not be added.

Ok, what's the reason they exist?

**Comment by weiji14** [comment]
> > > added
> > > pygmt/tests/baseline/test_plot3d_ogrgmt_file_multipoint_default_style_Path.png
> > > added
> > > pygmt/tests/baseline/test_plot3d_ogrgmt_file_multipoint_default_style_str.png
> > 
> > 
> > These two images are the same as `test_plot3d_ogrgmt_file_multipoint_default_style.png` and should not be added.
> 
> Ok, what's the reason they exist?

That one is a parametrized test where we're passing in either `str` or `Path`, and the output should be the same

https://github.com/GenericMappingTools/pygmt/blob/b31671eb88bd0cf202f059a78bc1907b06f9b786/pygmt/tests/test_plot3d.py#L432-L452

Maybe when [`pytest-mpl`](https://github.com/matplotlib/pytest-mpl) is creating the plots, it doesn't know to use the `filename="test_plot3d_ogrgmt_file_multipoint_default_style.png"`?

File: pygmt/tests/baseline/test_plot3d_ogrgmt_file_multipoint_default_style_str.png.dvc
Comment (weiji14) [inline pygmt/tests/baseline/test_plot3d_ogrgmt_file_multipoint_default_style_str.png.dvc:1]: @michaelgrund, you'll still need to keep one of these new files, but rename `test_plot3d_ogrgmt_file_multipoint_default_style_str` to `test_plot3d_ogrgmt_file_multipoint_default_style`.

**Pull request merged**